static/gsApp/components/navBillingStatus.spec.tsx

```diff
@@ -14,13 +15,8 @@ import SubscriptionStore from 'getsentry/stores/subscriptionStore';
 // Jun 06 2022
 const MOCK_TODAY = 1654492173000;
 
-jest.mock('moment-timezone', () => {
-  const moment = jest.requireActual('moment-timezone');
-  moment.now = jest.fn().mockReturnValue(1654492173000); // jest didn't like me using the constant
-  return moment;
-});
-
 describe('PrimaryNavigationQuotaExceeded', function () {
+  setMockDate(MOCK_TODAY);
```

Comment:
One more thing - can you put this in a `beforeEach()` and add an `afterEach` that calls `resetMockDate()`? Helps ensure that this doesn't leak